tests/kernel/threads/thread_apis/src/test_essential_thread.c

```diff
@@ -10,7 +10,11 @@
 
 struct k_thread kthread_thread;
 
+#ifdef CONFIG_COVERAGE_GCOV
+#define STACKSIZE 4096
+#else
 #define STACKSIZE 1024
```

Comment:
just add CONFIG_TEST_EXTRA_STACKSIZE instead

Reply:
In this test’s prj.conf , it defines a small CONFIG_TEST_EXTRA_STACKSIZE.

Reply:
that really shouldn't be set in prj.conf any clue why it is doing it?